README.md

```diff
@@ -5,45 +5,19 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://skimage.zulipchat.com)
 
 - **Website (including documentation):** [https://scikit-image.org/](https://scikit-image.org)
+- **Documentation:** [https://scikit-image.org/docs/stable/](https://scikit-image.org/docs/stable/)
 - **User forum:** [https://forum.image.sc/tag/scikit-image](https://forum.image.sc/tag/scikit-image)
 - **Developer forum:** [https://discuss.scientific-python.org/c/contributor/skimage](https://discuss.scientific-python.org/c/contributor/skimage)
 - **Source:** [https://github.com/scikit-image/scikit-image](https://github.com/scikit-image/scikit-image)
-- **Benchmarks:** [https://pandas.pydata.org/speed/scikit-image/](https://pandas.pydata.org/speed/scikit-image/)
 
-## Installation from binaries
+## Installation
 
 - **pip:** `pip install scikit-image`
 - **conda:** `conda install -c conda-forge scikit-image`
 
 Also see [installing `scikit-image`](INSTALL.rst).
```

Comment:
I don't think that will work, because we render this file as part of the documentation. Do you want to link to the built version? Feel free to just push to this PR.

Reply:
Huh, we do? I can only find the inclusion in our `pyproject.toml` but not where it is included in our generated HTML.

Reply:
I don't think we are currently rendering the README in the docs (although I could be wrong). I am going to commit the suggestion and merge this PR after the CI finishes.